src/test/scala/uk/gov/hmrc/emailaddress/EmailAddressFormatterSpecs.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2015 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emailaddress
+
+import org.joda.time.LocalDate
+import org.scalatest.{ Matchers, WordSpec }
+import play.api.data.FormError
+
+class EmailAddressFormatterSpecs extends WordSpec with Matchers {
+
+  "EmailAddress formatter" should {
+
+    "unbind correct Email" in {
+      EmailAddressFormatter.emailAddress().unbind("email", EmailAddress("example@example.com")) shouldBe Map("email" -> "example@example.com")
+    }
+
+    "create a mapping for a valid Email" in {
+      val result = EmailAddressFormatter.emailAddress().bind("email", Map("email" -> "example@example.com"))
+      result.isRight shouldBe true
```

Comment:
The tests would be clearer using http://www.scalatest.org/user_guide/other_goodies#eitherValues